docs/source/containerization/docker.mdx

```diff
@@ -81,11 +113,11 @@ docker run -p 4000:4000 \
   ghcr.io/apollographql/router:<image version>-debug
 ```
 
-The router will run under the control of heaptrack. The heaptrack output will be saved to the /data directory. The output can be analyzed directly using `heaptrack_gui` or `heaptrack_print` or shared with Apollo support staff.
+The router will run under the control of heaptrack. The heaptrack output will be saved to the `/data` directory. The output can be analyzed directly using `heaptrack_gui` or `heaptrack_print` or shared with Apollo support.
 
 ## Specifying the supergraph
 
-If we don't want to use uplink to retrieve our subgraph, we can manually specify the details.
+If we don't want to use [Apollo Uplink](/federation/managed-federation/uplink/) to retrieve our subgraph, we can manually specify the details.
```

Comment:
Can we add why anyone would want to do this?

Reply:
This is users who don't want to subscribe to our model for automatically updating the supergraph via uplink or who don't have connectivity (air-gapped) for their environment.